index.html

```diff
@@ -75,6 +75,6 @@ <h3>Past Activities</h3>
     </main>
 
   </body>
-  <script type="text/javascript" src="activity.js"></script>
-  <script type="text/javascript" src="main.js"></script>
+  <script type="text/javascript" src="./src/activity.js"></script>
```

Comment:
Maybe we should try again to make a seperate file for the queries and event listeners?

Reply:
Id say dont worry too much about that. as long as its all nicely organized in your main.js you are totally fine.